wordpress-mobile/WordPress-iOS
# Merge trial/nikola milicevic changes into develop

This merges the changes @nikola-milicevic made when he was on trial that were not merged into develop. 

WordPress Kit Related PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/443

## To test:
Follow the testing steps from: 
- https://github.com/wordpress-mobile/WordPress-iOS/pull/16629
- https://github.com/wordpress-mobile/WordPress-iOS/pull/16679

Also, [this](https://github.com/wordpress-mobile/WordPress-iOS/issues/16710) is the issue with related PRs that describes the redesigned UI for marking referrers as spam, that should be used for reference during testing.

## Regression Notes
1. Potential unintended areas of impact
The stats view referrer page structure was changed, this could affect how the users use the flow.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the flow.

3. What automated tests I added (or what prevented me from doing so)
Tests were added to the original PRs

PR submission checklist:
